cdc/api/open.go

```diff
@@ -795,16 +792,9 @@ func (h *openAPI) forwardToOwner(c *gin.Context) {
 
 	var owner *model.CaptureInfo
 	// get owner
-	err := retry.Do(ctx, func() error {
-		o, err := h.capture.GetOwnerCaptureInfo(ctx)
-		if err != nil {
-			log.Info("get owner failed, retry later", zap.Error(err))
-			return err
-		}
-		owner = o
-		return nil
-	}, retry.WithBackoffBaseDelay(300), retry.WithMaxTries(getOwnerRetryMaxTime))
+	owner, err := h.capture.GetOwnerCaptureInfo(ctx)
```

Comment:
Why do we remove retry here

Reply:
GetOwnerCaptureInfo already has an internal retry mechanism:
1. https://github.com/pingcap/tiflow/blob/master/cdc/capture/capture.go#L616-L621
2. https://github.com/pingcap/tiflow/blob/master/pkg/etcd/etcd.go#L223
3. https://github.com/pingcap/tiflow/blob/master/pkg/etcd/etcd.go#L559